airbyte-integrations/connectors/source-marketo/source_marketo/source.py

```diff
@@ -182,8 +184,12 @@ def stream_slices(self, sync_mode, stream_state: Mapping[str, Any] = None, **kwa
 
             export = self.create_export(param)
 
-            date_slice["id"] = export["exportId"]
-        return date_slices
+            status, export_id = export.get("status", "").lower(), export.get("exportId")
+            if status != "created" or not export_id:
```

Comment:
@davydov-d shouldn't this fail the sync instead of skipping silently? being in this position means we are missing data right?

Reply:
well, I'm not sure now.. it used to fail before this PR. I guess we need to make a retry, but it's impossible within the current architecture as the exception is thrown in the `stream_slices` method which is not retriable. I will create a follow-up ticket, if you're ok with that

Reply:
https://github.com/airbytehq/airbyte/issues/14149